### DISCUSSION TITLE
Ideas - 中文社区主站建设

### DISCUSSION THREAD

**Discussion by yufeiminds** [PECommunity/community#3] [Ideas]
设计文档（待补充），有什么需要讨论的可以在这里协作～

**Comment by node** [comment]
增加平台相关工作机会，接受社区成员投递 😄

**Reply by yufeiminds** [reply]
已添加～

**Comment by node** [comment]
站点内容架构
- 首页
    - 社区介绍
    - 概念介绍 
    - 动态
    - 成员
    - 如何加入 
-  工作机会 
-  本地社区
    - 北京 #loc-china-beijing
    - 上海 #loc-china-shanghai 
    - 深圳 #loc-china-shenzhen
    - 杭州 #loc-china-hangzhou

**Comment by yufeiminds** [comment]
站点架构改版讨论：

1. 使用 Nextra 和 MDX，便于基于 Markdown 扩展复杂的渲染能力，无需使用 JS 写落地页了。
2. 由于跨仓库的 Git Preview 能力构建比较复杂，考虑整合 website、glossary、mindmap 三个仓库到 community 仓库中，通过 [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) 文件来划分治理范围。
3. Markmap 的集成比较难，需要预构建为 SVG，无法实时渲染，是否考虑使用 mermaid mindmap 来代替
4. 需要补充：
    1. 成员信息
    2. 各栏目简介
5. 重新设计一个 icon，需要导出矢量版本
